screens/Signup.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { View, Text, StyleSheet, TouchableOpacity, ScrollView, Image } from "react-native";
+import { ButtonUi } from "../components/ui/Button";
+import { InputForm } from "../components/ui/InputForm";
+import { useNavigation } from '@react-navigation/native';
+import RadioGroup from 'react-native-radio-buttons-group';
+import { useMemo, useState } from "react";
+
+
+const SignUp = () => {
+
+  const radioButtons = useMemo(() => ([
+    {
+      id: '1', // acts as primary key, should be unique and non-empty string
+      label: 'อาจารย์ (@KMITL)',
+      value: 'professor'
+    },
+    {
+      id: '2',
+      label: 'นักเรียน (@KMITL)',
+      value: 'student'
+    },
+    {
+      id: '3',
+      label: 'ผู้เยี่ยมชม',
+      value: 'visitor'
+    }
+
+  ]), []);
+  const [selectedId, setSelectedId] = useState<string | undefined>();
+  return (
+    <View style={styles.screen}>
+      <ScrollView style={styles.card}>
+        <View style={styles.centeredContainer}>
+          <Image
+            style={{ marginBottom: 30 }}
+            source={require('../assets/Logo.png')}
+          />
+          <Text style={styles.signUpText}>
+            สมัครสมาชิก
+          </Text>
+        </View>
+        {/* <View style={{ justifyContent: 'center', flexDirection: 'row'}}> */}
+          <InputForm title={"ชื่อ *"} />
+          <InputForm title={"นามสกุล *"} />
+        {/* </View> */}
+        <InputForm title={"ชื่อบัญชีผู้ใช้ *"} />
+        <InputForm title={"เพศ *"} />
+        <InputForm title={"รหัสผ่าน *"} />
+        <InputForm title={"ยืนยันรหัสผ่าน *"} />
+        <InputForm title={"หมายเลขโทรศัพท์ *"} />
```

Comment:
```suggestion
         NameList.map(name => <InputForm title={name} />)
```
ควรนำไป map  map(name => <InputForm title={name} />)